docs/desktop/gui/theming.md

```diff
@@ -307,9 +305,9 @@ This example uses the (Monserrat font)[https://fonts.google.com/specimen/Montser
 
 Note that things like "family" are specified more than once and in different ways - the *top* level of properties (eg `size`, `family`, `bold`, etc relate to properties in the [qx.bom.webfonts.WebFont](apps://apiviewer/#qx.bom.webfonts.WebFont) class, whereas the properties of the `sources` key are slightly different.
 
-The top level properties are typically for defining what properties you would allow the browser to apply, and are analogous to the CSS properties.  For example, you can see that `family` is an array, because just like CSS the browser is to try and the first listed font family and then fallback to the second etc.
+The top level properties are typically for defining what properties you would allow the browser to apply, and are analogous to the CSS properties. For example, you can see that `family` is an array, because just like CSS the browser will try and the first listed font family and then fallback to the second etc.
```

Comment:
"will try the first" (vs "will try **and** the first")

Reply:
Done

Reply:
@derrell can you have a look again and approve?